crates/muvm/Cargo.toml

```diff
@@ -15,7 +15,7 @@ byteorder = { version = "1.5.0", default-features = false, features = ["std"] }
 env_logger = { version = "0.11.3", default-features = false, features = ["auto-color", "humantime", "unstable-kv"] }
 krun-sys = { path = "../krun-sys", version = "1.9.1", default-features = false, features = [] }
 log = { version = "0.4.21", default-features = false, features = ["kv"] }
-nix = { version = "0.28.0", default-features = false, features = ["user"] }
+nix = { version = "0.29.0", default-features = false, features = ["user"] }
```

Comment:
Should we move the shared dependencies versions back to `[workspace.dependencies]` in the root `Cargo.toml`?

/cc @slp :laughing: